### ISSUE TITLE
Logging out fails if the session is already timed out

### ISSUE THREAD

**Issue by garethbowen** [medic/medic-webapp#4673]
If I log out and the session is already invalid then nothing happens and there is an error in the console.

**Steps to reproduce**:
 - Log in
 - Wait for the session to timeout or call `DELETE [server]/_session` in another tab directly
 - Click the hamburger menu then the logout button
 - Confirm you want to logout

**What should happen**:
 - You are redirected to the login page

**What actually happens**:
 - The confirm modal disappears and you are left in the app and you get this in the browser console

```
DELETE http://localhost:5988/_session 401 (Unauthorized)
Possibly unhandled rejection: {"data":{"error":"unauthorized","reason":"Authentication required."},"status":401,"config":{"method":"DELETE","transformRequest":[null],"transformResponse":[null],"jsonpCallbackParam":"callback","url":"/_session","headers":{"Accept":"application/json, text/plain, */*"}},"statusText":"Unauthorized","xhrStatus":"complete"}
```

**Environment**:

- Instance: localhost
- Browser: Chrome, didn't test any others
- Client platform: Linux
- App: webapp
- Version: 3.0.0

**Comment by ngaruko** [comment]
LGTM. Tested on `alpha.dev`

**Comment by ngaruko** [comment]
I am seeing this happen again on `alpha.dev` with the following errors in the console:
![image](https://user-images.githubusercontent.com/6979995/42551563-f2416256-852b-11e8-843f-f2b48e72d4aa.png)

**Comment by garethbowen** [comment]
@vimemo Did you back port this to `3.0.x`?

**Comment by vimemo** [comment]
@garethbowen, yes. This fix was backported to 3.0.x.

**Comment by garethbowen** [comment]
Thanks! Here's the commit: https://github.com/medic/medic-webapp/commit/ffa8a6422b0139e79a80771997c47462a8aa7a4f
